auto_submit/lib/requests/check_pull_request.dart

```diff
@@ -39,14 +45,62 @@ class CheckPullRequest extends RequestHandler {
       return responses;
     }
     final List<Future<Response>> futures = <Future<Response>>[];
+    Set shouldMergeSet = <PullRequest>{};
```

Comment:
`final Set<PullRequest>`
([Explicitly type all list and map literals.](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-var-and-dynamic))